Sources/MiniPlayer/MiniPlayer.swift

```diff
@@ -14,4 +14,5 @@
 protocol MiniPlayer: VLCMiniPlaybackViewInterface, VLCPlaybackControllerDelegate {
     var contentHeight: Float { get }
     func updatePlayPauseButton()
+    var mediaService: MediaLibraryService? { get set }
```

Comment:
I think it would be better if it was directly passed to the specific player than adding it to the protocol.